src/hooks/useSeriesData.ts

```diff
@@ -11,18 +11,18 @@ type Data = {
 };
 
 export const useSeriesData = (
-  entityId?: string,
+  seriesId: string | undefined,
+  mediaId: string | undefined,
 ): {
   data: Data;
-  // isItemError: boolean;
   isPlaylistError: boolean;
   isLoading: boolean;
 } => {
-  const { data: seriesData, isLoading: isSeriesLoading, error: seriesError, isError: isSeriesError } = useSeries(entityId);
+  const { data: seriesData, isLoading: isSeriesLoading, error: seriesError, isError: isSeriesError } = useSeries(mediaId);
 
   const usePlaylistFallback = isSeriesError && seriesError?.code === 404;
```

Comment:
I think we should be able to decide whether to use the new or old series methods now without relying on a fallback on the error response.

Reply:
Well, we can only guess that in case there is `series` contentType, then we should use `series` api. But I am not sure whether it is safe enough to use this approach. In case contentType is different and we still need series api, it won't work.

One more thing here is that a media item can have both `series` contentType and old series flow (TBN for instance). This approach will let them to update smoothly because we will just fall back in case we don't find any series associated with a media item.